packages/remark-fix-guillemets/dist/index.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
```

Comment:
You should rebuild this file. Since we moved to jest it's not done before each test run anymore.